### PR TITLE
Add Development setup

### PR DESCRIPTION
**Pull request by rvsia**
closes https://github.com/data-driven-forms/react-forms/issues/142

**Description**

- Adds `README` for each repo with customized content (most of it is copy/paste however)
- Adds `Development setup` to online documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -154,6 +158,77 @@ Please use our [documentation site](https://data-driven-forms.org/). In case of
...
+Types:
+- `feat`: a new feature, will trigger new `_.X._` release
+- `fix`: a fix, will trigger new `_._.X` release
+
+Packages:
+- Please describe which package is being changed `pf3`, `renderer`, ...
+
+Please, do not use Semantic Release, if you update only the demo.
+
+All packages are releasing together and they share the version number.
```
Comment (Hyperkid123) [inline README.md]: I would maybe add an examaple of a commit message here

**Comment by Hyperkid123** [comment]
@skateman is there something else you would like to see included?

File: README.md (hunk 1 above)
Comment (rvsia) [reply README.md]: It is a few lines above: `fix(pf3): title accepts node`

**Comment by brumik** [comment]
Sorry, slipped my attention, but I think it is good enough. Thanx :)

**Pull request merged**